ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -37879,6 +37879,111 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: '@yearly'
+  decorate: true
+  decoration_config:
+    timeout: 6h0m0s
+  name: periodic-ci-openshift-release-master-nightly-4.12-install-analysis-gcp-ovn-ipi
```

Comment:
I forgot what we said about creating install check for blocking jobs. Any job variants covered by blocking jobs already guarantee install success. Let's sync with the team on Monday.